File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/InlineObject.scala

package org.openapitools.client.model


case class InlineObject (
    _addresses: List[AddressesAddresses]
)
object InlineObject {
    def toStringBody(var_addresses: Object) =
        s"""
        | {
        | "addresses":$var_addresses
        | }
        """.stripMargin
}

File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/InlineObject.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.AddressesAddresses
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param addresses 
 */
data class InlineObject (

        @get:NotNull 
        @JsonProperty("addresses") val addresses: List<AddressesAddresses>
) {

}

